here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/AbstractEventHandler.java

```diff
@@ -46,7 +48,7 @@ public AbstractEventHandler(final @NotNull INaksha hub) {
   }
 
   protected @NotNull Result validateWritePluginRequest(final @NotNull Plugin<?, ?> plugin) {
-    if (plugin.getClassName().isEmpty()) {
+    if (plugin.getClassName() == null || plugin.getClassName().isEmpty()) {
```

Comment:
.getClassName() is guaranteed @NotNull, so it's fine with the original

Reply:
Unfortunately that's not the case. JetBrain's `NotNull` annotation is only for IDE's highlighting - it does not affect compile or runtime in any way, there's no actual validation going on.

I have tested it (`tc0062`) - the original code (without null check) does not handle missing `className` and causes exception on the server side (resulting in `500` http response code).
So this has to stay this way

Reply:
I think the resource files for test case 0062 are missing

Reply:
> I think the resource files for test case 0062 are missing ;)

Thanks, somehow I managed to not include them in staging 🙈 

Regarding the rest of your concerns - we've discussed is on webex